profile/src/main/java/rdx/works/profile/data/repository/PublicKeyProvider.kt

```diff
@@ -7,12 +7,13 @@ import rdx.works.profile.data.model.extensions.nextAccountIndex
 import rdx.works.profile.data.model.factorsources.DerivationPathScheme
 import rdx.works.profile.data.model.factorsources.DeviceFactorSource
 import rdx.works.profile.data.model.factorsources.FactorSource
+import rdx.works.profile.data.model.factorsources.Slip10Curve
 import rdx.works.profile.data.model.pernetwork.DerivationPath
 import rdx.works.profile.derivation.model.KeyType
 import rdx.works.profile.derivation.model.NetworkId
 import javax.inject.Inject
 
-class AccessFactorSourcesProvider @Inject constructor(
+class PublicKeyProvider @Inject constructor(
```

Comment:
Like @micbakos-rdx noticed in my previous PR this class maybe it doesn't make too much sense? 
I could move these function in `MnemonicRepository`? Or create extension functions (on top of what?)?

Please add your thoughts on this.